rocketpy/EnvironmentAnalysis.py

```diff
@@ -786,10 +865,67 @@ def animate_wind_gust_distribution_over_average_day(self):
         """Animation of how the wind gust distribution varies throughout the day."""
         ...
 
-    # TODO: Implement
+    # TODO: Create tests
     def animate_wind_profile_over_average_day(self):
         """Animation of how wind profile evolves throughout an average day."""
-        ...
+        # Gather animation data
+        altitude_list = np.linspace(1495.155, 12000, 100)  # TODO: parametrize
```

Comment:
Can't you use `self.elevation` instead of 1495.155? I'm try to understand this part

Reply:
Thanks for the tip boss! I have implemented it in my latest commit.